### DISCUSSION TITLE
对于返回类型里包含泛型的数据，处理不友好。

### DISCUSSION THREAD

**Discussion by SpaceMiao** [Robot-L/translator#14] [General]
比如Controller返回是泛型类型
```
RestResponse<T> {
     private String code;
     private String message;
     private  T data;
}
```
你的代码里面`com.robot.transform.component.TransformClass#getClassFromField`方法在处理属性`data`时，获取到的是`Class<Object>`，这时候数据就全乱了。
之前我在实现类似的功能的时候，发现这个问题始终无法解决，最后就放弃了。

**Comment by luo-zhan** [comment]
这个要用解包器功能去做的，看下WIKI[解包器](https://github.com/luo-zhan/Transformer/wiki/%E8%A7%A3%E5%8C%85%E5%99%A8-UnWrapper#%E8%87%AA%E5%AE%9A%E4%B9%89%E8%A7%A3%E5%8C%85%E5%99%A8)的[自定义解包器]一节
